app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,22 @@
 package com.codesoom.assignment;
 
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+    private static final int port = 8000;
```

Comment:
JAVA에는 `static final` 변수를 대문자 스네이크 케이스로 작성하는 관례가 있습니다. 즉 이 변수는 대문자로 다음과 같이 작성하는 것이 좋겠죠.

```suggestion
    private static final int PORT = 8000;
```

camelCase로 작성한 변수라면 `_`를 추가한 snake_case로 작성하는데요, 만약 변수 이름이 `serialPort` 였다면 `SERIAL_PORT` 처럼 쓴다고 생각하시면 됩니다.

Reply:
![image](https://user-images.githubusercontent.com/41536271/105133074-61f57980-5b2f-11eb-8369-f9819126e067.png)

변경하니까 추천 리스트에 대문자로 나오는군요! 감사합니다!

Reply:
인텔리제이의 리팩토링 기능을 잘 활용하시는 것 같아 기쁩니다.